README.md

```diff
@@ -1,12 +1,39 @@
-# Fonts sourced from Apple
+# San Francisco Fonts
 
-This flake provides easy access to the Apple fonts listed on their website. Just add it as a font package, then configure your programs to use it.
+> [!Note]
+> The fonts are sourced directly from Apple, via [apple.com](https://apple.com).
+> Add this flake as a font package, then configure your programs to use it.
 
-Packages are also provided that automatically patch the fonts with the [nerd font patcher](https://github.com/ryanoasis/nerd-fonts). This provides a font that includes the most common symbols, suitable for use in the terminal, statusbars, etc.
+>[!Tip]
+> Packages are also provided that automatically patch the fonts with the [nerd font patcher](https://github.com/ryanoasis/nerd-fonts). This provides a font that includes the most common symbols, suitable for use in the terminal, statusbars, etc.
 
-# Example
+## CONFIGURATION EXAMPLE
 
-In my case, I use [stylix](https://stylix.danth.me/):
+```nix
+{
+  description = "Configuration Demonstration";
+  inputs.nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+
+  # San Francisco Fonts | Apple Fonts
+  inputs.apple-fonts.url= "github:Lyndeno/apple-fonts.nix";
+  inputs.apple-fonts.inputs.nixpkgs.follows = "nixpkgs";
```

Comment:
```suggestion
```

Let's not recommend this. For other flakes, I would do this, but the resulting package of this flake is just font files that shouldn't really change across nixpkgs versions (unless the font patcher OR it's dependencies are changed).

Having this follows causes the fonts to rebuild more often, and it can take a while. What are your thoughts?